package.json

```diff
@@ -683,6 +683,16 @@
                 "type": "object",
                 "description": "Environments variables override in Visual Studio Code launched ESP-IDF Debug Adapter",
                 "default": null
+              },
+              "initGdbCommands": {
+                "type": "array",
+                "description": "%esp_idf.initGdbCommands.description%",
+                "items": {
+                  "type": "string",
+                  "description": "%esp_idf.debuggers.text.description%",
+                  "default": ""
+                },
+                "default": []
```

Comment:
[Question]

Is this required for the gdb-stub ?

Reply:
As far as I know we need to set `target remote $port` to gdbstub to work. The actual setting should not be required in the launch.json